config/bootstrap.go

```diff
@@ -0,0 +1,88 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/multiformats/go-multiaddr"
+)
+
+const (
+	defaultMinimumPeers     = 1
+	defaultBootstrapPeriod  = 30 * time.Second
+	defaulConnectionTimeout = defaultBootstrapPeriod / 3
+)
+
+// defaultBootstrapAddresses are the hardcoded bootstrap addresses.
+var defaultBootstrapAddresses = []string{}
```

Comment:
are you planning to load this in via the `go build` or some other way?

Reply:
No.  This is just a place holder for if/when we want to hardcode some bootstrap addresses as IPFS does.

Reply:
we should find a way to default to the filecoin bootstrap nodes, probably

Reply:
namely we should use https://github.com/filecoin-project/lotus/blob/7a55e8e892a2ed7a76ae4a75c1e33580688c3d0e/build/bootstrap/mainnet.pi

Reply:
Added hardcoded values.  These are written into the config file during `./storetheindex init`